### PR TITLE
fix(i18n): fix broken header

### PR DESCRIPTION
**Pull request by ykzts**
same as #1305

### CODE REVIEW

File: src/lang/ja.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
   "collective.category.studentclub": "Student Club",
   "collective.changeUrl.label": "URL slug",
   "collective.collective.memberOf.collective.parent.subtitle": "{n, plural, one {this collective is} other {{n} collectives are}} part of our collective",
-  "collective.collective.memberOf.collective.parent.title": "{n, plural, one {this collective is} other {{n} collectives are}} part of our collective",
+  "collective.collective.memberOf.collective.parent.title": "Member collectives",
```
Comment (ykzts) [inline src/lang/ja.json]: image

File: src/components/GoalsCover.js [hunk 2]
```diff
@@ -502,11 +503,15 @@ class GoalsCover extends React.Component {
...
                 id="cover.budget.text"
-                defaultMessage="Thanks to your financial contributions, we are operating on an estimated annual budget of"
+                defaultMessage="Thanks to your financial contributions, we are operating on an estimated annual budget of {yearlyBudget}"
+                values={{
+                  yearlyBudget: (
+                    <span className="annualBudget">
+                      {formatCurrency(get(collective, 'stats.yearlyBudget'), collective.currency, { precision: 0 })}
+                    </span>
+                  ),
+                }}
```
Comment (ykzts) [inline src/components/GoalsCover.js]: image

**Comment by Betree** [comment]
Thanks @ykzts!

I'm going to add some lines in our [documentation](https://docs.opencollective.com/help/developers/translations) for us to take to good habit of using strings like `"Pending approval from {host}"` instead of `"Pending approval from"` to acknowledge the fact that words order will not be preserved across languages!

Great job fixing those keys. Also you can now use https://crowdin.com/project/opencollective/ja that is way more convenient to use (when the keys are correct :wink:)

**Comment by ykzts** [comment]
image

my works 👍

**Comment by Betree** [comment]
@ykzts Wow that's amazing! I'll merge all these very soon.

Oh and also, you're now a member of @opencollective/translators so feel free to push directly on this repository if you want without the need to maintain a separate fork (only `master` branch is protected)

**Comment by ykzts** [comment]
> Oh and also, you're now a member of @opencollective/translators so feel free to push directly on this repository if you want without the need to maintain a separate fork (only master branch is protected)

thx! ❤️

**Pull request merged**